cli/jobs/pipelines/automl/image-instance-segmentation-task-fridge-items-pipeline/components/component_preprocessing.yaml

```diff
@@ -0,0 +1,26 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+name: data_preprocessing
+display_name: Data Preprocessing
+version: 1
+type: command
+inputs:
+  train_data:
+    type: mltable
+  validation_data:
+    type: mltable
+outputs:
+  preprocess_train_data:
+    type: mltable
+  preprocess_validation_data:
+    type: mltable
+code: ./src/
+environment:
+  name: pipeline-custom-environment
+  image: mcr.microsoft.com/azureml/openmpi3.1.2-ubuntu18.04
+  conda_file: ../environment/preprocessing_env.yaml
+command: >-
+  python preprocess.py
```

Comment:
can you please provide some context on why we need to add this pre-processing component for image tasks? This component isn't doing anything for images and will probably cause confusion for customers. Currently images doesn't support any data pre-processing...

Reply:
Done..Removed for vision !!